nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -397,6 +397,8 @@
 - `security.pam.u2f` now follows RFC42.
   All module options are now settable through the freeform `.settings`.
 
+- A module for foot terminal is added.
```

Comment:
Please move it to the "New Services" section.

Reply:
Is a new program considered as a new service? :)